atoma-event-subscribe/sui/src/subscriber.rs

```diff
@@ -213,7 +220,24 @@ impl SuiSubscriber {
     #[instrument(skip(self, event_data))]
     async fn handle_prompt_event(&self, event_data: Value) -> Result<(), SuiSubscriberError> {
         debug!("event data: {}", event_data);
-        let request = Request::try_from((self.id, event_data))?;
+        let mut request = Request::try_from((self.id, event_data))?;
+        let metadata = AtomaInputMetadata {
+            user_id: request.params().prompt(),
+            ticket_id: hex::encode(request.id().as_slice()),
+            node_id: self.id,
+            input_source: atoma_types::InputSource::Firebase,
+            input_type: atoma_types::InputType::Text,
+        };
+        let (oneshot_sender, oneshot_receiver) = tokio::sync::oneshot::channel();
+        self.input_manager_tx
+            .send((metadata, oneshot_sender))
+            .await
+            .map_err(Box::new)?;
+        let result = tokio::time::timeout(Duration::from_secs(5), oneshot_receiver)
```

Comment:
Let's hardcode the duration in a `const`. Also, isn't this introducing too much latency ? I wonder if we would not be good with a few ms ?

Reply:
we can finetune this with testing, but I would prefer for the alpha version more timeout instead of failing.